kube-client/Cargo.toml

```
@@ -12,10 +12,11 @@ keywords = ["kubernetes", "client",]
 categories = ["web-programming::http-client", "network-programming", "api-bindings"]
 
 [features]
-default = ["client"]
+default = ["client", "ring"]
```

```suggestion
default = ["client", "rustls-tls", "ring"]
```

possibly worth listing this explicitly